html/apps/tweetfeed/css/style.css

```diff
@@ -0,0 +1,172 @@
+body {
+  background-color: gray !important;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2597868539/issues/source?bid=3288751&fileBranchId=3307409#l2)